engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/DevProcessConfigCreator.scala

```diff
@@ -52,14 +57,19 @@ class DevProcessConfigCreator extends ProcessConfigCreator {
 
   private def kafkaConfig(config: Config) = KafkaConfig.parseConfig(config)
 
-  override def sinkFactories(processObjectDependencies: ProcessObjectDependencies): Map[String, WithCategories[SinkFactory]] =
+  override def sinkFactories(processObjectDependencies: ProcessObjectDependencies): Map[String, WithCategories[SinkFactory]] = {
+    val confluentFactory: ConfluentSchemaRegistryClientFactory = if (processObjectDependencies.config.hasPath(DevProcessConfigCreator.emptyMockedSchemaRegistryProperty)) {
```

Comment:
What will happend when `withMockedConfluent: false` in config?

Reply:
ok, I made it a bit better ;)